components/Account/Account.jsx

```diff
@@ -3,13 +3,16 @@ import { useUser, useSupabaseClient } from "@supabase/auth-helpers-react"
 import styles from "./Account.module.css"
 import cn from "classnames"
 import Avatar from "@/components/Avatar/Avatar"
+import Releases from "../Releases/Releases"
+import CreateRelease from "../Releases/CreateRelease"
 
 export default function Account({ session }) {
   const supabase = useSupabaseClient()
   const user = useUser()
   const [loading, setLoading] = useState(true)
   const [username, setUsername] = useState(null)
   const [avatarUrl, setAvatarUrl] = useState(null)
+  const [createNewRelease, setCreateNewRelease] = useState(false)
```

Comment:
I commented somewhere else about this (a little out of order) but now I see where this is being defined. What exactly is this doing? It seems like it just toggles a boolean but the naming makes me think it "creates a new release"